### ISSUE TITLE
Enhancement Request: Export QRDA without a db connection

### ISSUE THREAD

**Issue by jkotanchik-SB** [projecttacoma/cqm-reports#161]
The MADiE Team is working on our QRDA Export feature for QDM measures and was hoping to use cqm-reports exposed as a web service. 

We tried out a PoC and got stuck when we realized its export feature has a runtime dependency on the Bonnie database. Such that, without an active connection to retrieve measures, it will error out.

When an Array of `CQM::Measure` is passed in as the `measures` argument into:

    Qrda1R5.new(patient, measures, options).render

We get an error originating from `qrda-export/helper/view_helper.rb` on line 7, with the following message:

    undefined method `only' for [#<CQM::Measure ...

Since `only` comes from the Mongoid library, replacing the Array of `CQM::Measure` with `Mongoid::Critiera` results in a successful QRDA render. 

This creates a dependency on the QRDA Exporter to have an active connection with Bonnie's db. In order for MADiE to utilize the QRDA Export functionality, we request a feature be added that enables QRDA Export without any database dependency.

**Comment by dczulada** [comment]
@jkotanchik-SB 

Can you see if this branch suits your needs
https://github.com/projecttacoma/cqm-reports/tree/measure_objects

This branch is currently based upon another branch (recent_updates) that will merge in recent changes we've made to a cypress specific branch into the master branch.

**Comment by jkotanchik-SB** [comment]
@dczulada 

Thanks for the tip! That branch has been working well for my little proof-of-concept.

I was able to successfully export a QRDA with measure and patient objects, though I did need the mongoid dependency and an active mongo connection to make the QRDA exporter work. I'm guessing that's more to do with `cqm-models` as having a connection open to my local madie mongo was enough to generate the QRDA export.

I didn't quite follow whether the `measure_objects` branch will be merged to master or the branch it's based on, `recent_updates`, would be merged to master. And since it worked well, I'm curious to know, will the option of using measure objects be supported longterm?
